install_config/topics/ocp_osp_dns.adoc

```diff
@@ -0,0 +1,45 @@
+==== Setting up DNS for {product-title}
+
+DNS service is an important component in the {product-title} environment. Regardless of 
+the provider of DNS, an organization is required to have certain records in 
+place to serve the various {product-title} components.
+
+WARNING: Using `/etc/hosts` is not valid, a proper DNS service must exist.
```

Comment:
[WARNING]
    ====
    Using `/etc/hosts` is not valid. A proper DNS service must exist.
    ====